### ISSUE TITLE
Gravity Forms Entry export fails on sites with multiple replicas

### ISSUE THREAD

**Issue by curtismchale** [proudcity/wp-proudcity#2454]
**Build**
- [staging](https://console.cloud.google.com/cloud-build/builds;region=global/30c7a0b3-36f3-4af8-a697-4dc743c9fa23?project=proudcity-1184)
- [master](https://console.cloud.google.com/cloud-build/builds;region=global/07a195fa-fcfd-42e8-80bd-cdd90c2d4528?project=proudcity-1184)

## Source

- CM
- [Notes](https://github.com/proudcity/developers/blob/main/Github%20Issue%20Notes/2454%20-%20Gravity%20Forms%20Export.md)

## Describe the bug
When we add more than 1 replica to a site the [gravity forms entry export fails](https://www.santa-ana.org/wp-admin/admin.php?page=gf_export).

## To reproduce
1. Go to https://www.santa-ana.org/wp-admin/admin.php?page=gf_export
2. Click on any form export
3. export form
4. see that no file downloads

## Expected behavior
The number of replicas should not affect the ability of GF to export form entries.

[See GF docs on testing export functions](https://docs.gravityforms.com/resolving-issues-gravity-forms-exports/). They say it uses an AJAX query, so are AJAX queries in general broken when we have 2 replicas?

## QA

Notes:

* 

Links:

*

**Comment by kevindherman** [comment]
@curtismchale this [support ticket](https://proudcity.zendesk.com/agent/tickets/6786) needs access to their imports. Can you help them out if the platform fix isn't ready?

**Comment by curtismchale** [comment]
Based on my research yesterday (research notes linked above), the file is created as expected. I added 2 replicas to `beta` and then created an export, which was blank. I made note of the filename that showed up in error dialogues and then when I reduced to 1 replica I found the file exactly where it was expected to be.

I wonder if it's getting created on the 2nd pod that I'm not currently using instead of the pod I'm actively using.
